pkg/providers/tinkerbell/validator_test.go

```diff
@@ -75,3 +150,15 @@ func newHardwareConfigWithHardware(hardwareCount int) hardware.HardwareConfig {
 		Hardwares: make([]tinkv1alpha1.Hardware, hardwareCount),
 	}
 }
+
+func newValidTinkerbellDatacenterConfig() *v1alpha1.TinkerbellDatacenterConfig {
+	return &v1alpha1.TinkerbellDatacenterConfig{
+		Status: v1alpha1.TinkerbellDatacenterConfigStatus{},
+		Spec: v1alpha1.TinkerbellDatacenterConfigSpec{
+			TinkerbellIP:           "1.1.1.1",
+			TinkerbellCertURL:      "1.1.1.1",
```

Comment:
typically the cert url is of the format `http://hostname:port/cert`

Reply:
Done.